### ISSUE TITLE
Lint whitespace on object spread

### ISSUE THREAD

**Issue by adidahiya** [palantir/tslint#2122]
### Config

New option on `whitespace` rule called "check-spread"

### Failing code

```ts
const thing = {
    ... otherThing,
};
```

### Passing code

```ts
const thing = {
    ...otherThing,
};
```
